test/evergreen/run_format_configs.sh

```diff
@@ -10,7 +10,7 @@ set -u
 cd $(git rev-parse --show-toplevel)
 
 # Walk into the test/format directory
-cd build_posix/test/format
+cd cmake_build/test/format
```

Comment:
Should we `set -e` to exit if this command fails? Or even the first `cd` line 10?

Reply:
Done: e9e9a26faf8cb53cb02a27a864f5b894a2d1d6b2
I did it just around these commands. The test intends to catch failures in the subsequent format runs.